### PR TITLE
New section about JSON file manifest strategy

### PR DESCRIPTION
**Pull request by jeremyjumeau**
@see https://symfony.com/blog/new-in-symfony-3-3-manifest-based-asset-versioning

### CODE REVIEW

File: components/asset.rst [hunk 1]
```diff
@@ -139,6 +139,29 @@ string as the second argument of the ``StaticVersionStrategy`` constructor::
     echo $package->getUrl('image.png');
     // result: v1/image.png
 
+JSON File Manifest
+..................
+
+Use the :class:`Symfony\\Component\\Asset\\VersionStrategy\\JsonManifestVersionStrategy`
+to take advantage of the strategy used by popular tools such as `Webpack 
+<https://webpack.js.org/>`_, which generate a JSON file mapping all source file names to their corresponding output file. For example::
```
Comment (tomtomau) [inline components/asset.rst]: which **generates**

**Review by tomtomau — CHANGES_REQUESTED** [review]
Looks great - thanks for your contribution!

Could you additionally update [frontend/encore/versioning.rst](https://github.com/jeremyjumeau/symfony-docs/blob/master/frontend/encore/versioning.rst) to link through to this section to "read more" about asset versioning?

I know when I was getting started with Encore it would have been helpful 👍

**Comment by jeremyjumeau** [comment]
Thanks for your feedback! I will update the "learn more" section of Encore Component too!

**Review by jeremyjumeau — COMMENTED** [review]
@tomtomau do you think we should add a link to *Encore Versioning* documentation in the asset's doc "Learn More" section?

```
Learn more
----------

* :doc:`frontend/encore/versioning`
```

**Review by javiereguiluz — APPROVED** [review]

**Pull request closed without merging**